lang/python/python3-pyinotify/Makefile

```
@@ -0,0 +1,44 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pyinotify
+PKG_VERSION:=0.9.6
+PKG_RELEASE:=1
+PKGARCH:=ALL
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/pyinotify/
+PKG_HASH:=9c998a5d7606ca835065cdabc013ae6c66eb9ea76a00a1e3bc6e0cfe2b4f71f4
+
+PKG_MAINTAINER:=Gerald Kerma <gandalf@gk2.net>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+#include $(TOPDIR)/feeds/packages/lang/python/pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include $(TOPDIR)/feeds/packages/lang/python/python3-package.mk
```

Including or rather hardcoding this way: `$(TOPDIR)/feeds/packages` is wrong. Because you count that `packages` folder is not renamed anyhow. By default, it is correct,but you dont count with individual configurations done in feeds.conf. E.g. Github Actions is using [packages-ci](https://github.com/openwrt/packages/blob/a592e95dc12f3105345d8eb51304dbf2bfa22633/.github/workflows/multi-arch-test-build.yml#L67)

The better solution is to use:
include ../pypi.mk
```suggestion
include ../pypi.mk
include $(INCLUDE_DIR)/package.mk
include ../python3-package.mk
```

Because this Makefile and all these includes are in one shared/common folder